src/core/Akka.MultiNodeTestRunner.AzureDevOps/Akka.MultiNodeTestRunner.AzureDevOps.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\common.props" />
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkLibVersion);$(NetStandardLibVersion)</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.TeamFoundationServer.Client" Version="16.153.0" />
```

Comment:
I'd avoid the route of creating another NuGet package here. We just need TRX output formatting, not necessarily a tool for connecting directly to Azure DevOps.